xmodule/assets/video/public/js/04_video_control.js

```diff
@@ -0,0 +1,164 @@
+        state.videoControl.fadeOutTimeout = state.config.fadeOutTimeout;
+
+        state.videoControl.el.addClass('html5');
+        state.controlHideTimeout = setTimeout(state.videoControl.hideControls, state.videoControl.fadeOutTimeout);
+    }
+}
+
+// function _bindHandlers(state)
+//
+//     Bind any necessary function callbacks to DOM events (click, mousemove, etc.).
+function _bindHandlers(state) {
+    if ((state.videoType === 'html5') && (state.config.autohideHtml5)) {
+        state.el.on({
+            mousemove: state.videoControl.showControls,
+            keydown: state.videoControl.showControls
+        });
+    }
+
+    if (state.config.focusFirstControl) {
+        state.el.on('initialize', state.videoControl.focusFirst);
+    }
+    state.el.on('destroy', state.videoControl.destroy);
+}
+
+// ***************************************************************
+// Public functions start here.
+// These are available via the 'state' object. Their context ('this' keyword) is the 'state' object.
+// The magic private function that makes them available and sets up their context is makeFunctionsPublic().
+// ***************************************************************
+
+function focusFirst() {
+    this.videoControl.el.find('.vcr a, .vcr button').first().focus();
+}
+
+function show() {
+    this.videoControl.el.removeClass('is-hidden');
+    this.el.trigger('controls:show', arguments);
+}
+
+function showControls(event) {
+    if (!this.controlShowLock) {
+        if (!this.captionsHidden) {
+            return;
+        }
+
+        this.controlShowLock = true;
+
+        if (this.controlState === 'invisible') {
+            this.videoControl.el.show();
+            this.controlState = 'visible';
+        } else if (this.controlState === 'hiding') {
+            this.videoControl.el.stop(true, false).css('opacity', 1).show();
+            this.controlState = 'visible';
+        } else if (this.controlState === 'visible') {
+            clearTimeout(this.controlHideTimeout);
+        }
+
+        this.controlHideTimeout = setTimeout(this.videoControl.hideControls, this.videoControl.fadeOutTimeout);
+        this.controlShowLock = false;
+    }
+}
+
+function hideControls() {
+    let _this = this;
+
+    this.controlHideTimeout = null;
+
+    if (!this.captionsHidden) {
+        return;
+    }
+
+    this.controlState = 'hiding';
+    this.videoControl.el.fadeOut(this.videoControl.fadeOutTimeout, function() {
+        _this.controlState = 'invisible';
+        // If the focus was on the video control or the volume control,
+        // then we must make sure to close these dialogs. Otherwise, after
+        // next autofocus, these dialogs will be open, but the focus will
+        // not be on them.
+        _this.videoVolumeControl.el.removeClass('open');
+        _this.videoSpeedControl.el.removeClass('open');
+
+        _this.focusGrabber.enableFocusGrabber();
+    });
+}
+
+function updateVcrVidTime(params) {
```

Comment:
This method's logic do not seems completely sync with the previous file in edx-platform
https://github.com/openedx/edx-platform/blob/a11086ffac05f8296937d63a4ecf387e5d450792/xmodule/js/src/video/04_video_control.js#L154

Is this intentional update?